src/RadhydroPulseMGint/test_radhydro_pulse_MG_int.cpp

```diff
@@ -0,0 +1,551 @@
+/// \file test_radhydro_pulse_MG_int.cpp
+/// \brief Defines a test problem for multigroup radiation in the diffusion regime with advection by gas using group-interpolated variable opacity.
+///
+
+#include "test_radhydro_pulse_MG_int.hpp"
+#include "AMReX_BC_TYPES.H"
+#include "AMReX_Print.H"
+#include "RadhydroSimulation.hpp"
+#include "fextract.hpp"
+#include "physics_info.hpp"
+#include "planck_integral.hpp"
+#include "radiation_system.hpp"
+
+struct MGintProblem {
+};		     // dummy type to allow compile-type polymorphism via template specialization
+struct GreyProblem { // Advecting grey pulse
+};
+
+AMREX_GPU_MANAGED double spec_power = -1.0; // NOLINT
```

Comment:
We discussed issues with hard-coding this value. Has there been progress on that?

Reply:
Here spec_power = -1 is set for testing purpose. It gives a solution very close to the exact one, so I can compare it with the exact solution and set a reasonable error_tol. Currently, a free power-law slope would produce a result with a much higher error when nGroup <= 4, which is what I use for testing. In a future PR, I will try to work out a volume-averaged Piecewise power law method.